src/main/ansible/vagrant.yml

```diff
@@ -23,29 +23,31 @@
     easy_test_users_email: "{{ deployer_email }}"
     easy_test_users_ldapadmin_password: "{{ easy_ldap_dir_ldapadmin_password }}"
   roles:
+    - dans.capture-deployer-email
     - dans.local-test-vm-base
+    - dans.local-yum-repo
     - dans.easy-ldap-dir
-    - dans.capture-deployer-email
     - dans.easy-test-users
-    - dans.local-yum-repo
 
 - hosts: "test"
   become: yes
   tasks:
-    - name: Installing package
-      yum:
-        name: dans.knaw.nl-easy-sword2
-        state: latest
+  - name: Create directories
+    file:
+      path: "{{ item }}"
+      recurse: yes
+      state: directory
+      mode: "0777"
+    with_items:
```

Comment:
I'm not. I think both configuration of these directories and creating them should be left to the deployer, e.g. in ansible/vagrant. Imagine the deployer wishes to put these directories in a different place, now he first has to remove the old directory (which he might not have the rights to?), then create the new directory and set the configurations correctly. I think we should keep the rpm install for what it currently does: stopping the service, installing the software and restarting the service.